src/app/diff-view/diff-view.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnChanges, Input } from '@angular/core';
+import { diffJson } from 'diff'
+
+@Component({
+  selector: 'diff-view',
+  templateUrl: './diff-view.component.html',
+  styleUrls: ['./diff-view.component.scss']
+})
+export class DiffViewComponent implements OnChanges {
+  @Input() oldObject: object;
+  @Input() newObject: object;
+  diffObjects: Object[];
```

Comment:
use the correct type,(as far as I remember it's `IDiffResult`, you can import from `diff`)